rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -883,4 +890,136 @@ impl SDKClient {
         let cancels = self.parse_cancels(&signature).await;
         Some((signature, cancels))
     }
+
+    // Useful if unsure trader has an ATA for the base or quote or has a seat on the market
+    pub async fn get_limit_order_new_maker_ixs(
+        &self,
+        market_key: &Pubkey,
+        price: u64,
+        side: Side,
+        size: u64,
+    ) -> anyhow::Result<Vec<Instruction>> {
+        let metadata = self.get_market_metadata(market_key);
+        let base_ata_instruction_option = create_ata_ix_if_needed(
```

Comment:
Hm what do you think this should look like?

In my mind, the flow is if you're unsure if you have the right ATAs and seats, such as upon initialization, you can call this method. If you are sure, such as on subsequent usage, then use `get_limit_order_existing_maker_ixs`

Reply:
this is fine, but there should then be an example on how to send limit orders after the accounts have been set up

Reply:
I think this probably shouldn't include the place limit order instruction. Just the setup instructions. So the flow is:

- provision account
- set up seat, ata
- place and cancel orders to your hearts desires

Reply:
Ahh gotcha. So I originally included the check/create seat instruction inside the basic get order ix because a trader may have been evicted in between orders. If we remove check/create seat, are we okay with the order failing in that scenario and having the trader check it / handle it?

Reply:
That structure makes sense to me.

In the newest commit [416e0e6](https://github.com/Ellipsis-Labs/phoenix-sdk/pull/87/commits/416e0e68faf2260ac58c89521b20d67ad3a97dbc), we now have the following instruction builder functions:
- provision ATAs if needed
- check & claim seat (including evict), if needed in `create_claim_seat_ix_if_needed`
- `get_maker_setup_instructions_for_market` which does both of the above
-  new maker limit order which uses the maker setup instructions function and adds a limit order
- new maker post only order which uses the maker setup instructions function and adds a post only order